draft-ietf-httpbis-connect-tcp.md

```diff
@@ -168,10 +168,13 @@ When closing connections, endpoints are subject to the following requirements:
 * When a TCP connection reaches the TIME-WAIT or CLOSED state, the associated endpoint MUST close its send stream.
   - If the connection closed gracefully, the endpoint MUST close the send stream gracefully.
   - Otherwise, the endpoint SHOULD close the send stream abruptly, using a mechanism appropriate to the HTTP version:
-    - HTTP/3: RESET_STREAM with H3_CONNECT_ERROR
-    - HTTP/2: RST_STREAM with CONNECT_ERROR
-    - HTTP/1.1 over TLS: a TLS Error Alert
-    - HTTP/1.1 (insecure): TCP RST.
+    - HTTP/3: reset the stream with H3_CONNECT_ERROR;
+      see {{!RFC9000, Section 19.4}} and {{?RFC9114, Section 8.1}}
+    - HTTP/2: reset the stream with CONNECT_ERROR;
+      see {{!RFC9113}}, Sections 6.4 and 7
+    - HTTP/1.1 over TLS: TCP shutdown without a TLS closure alert;
```

Comment:
```suggestion
    - HTTP/1.1 over TLS: TCP shutdown, with or without a TLS closure alert;
```

Reply:
As noted above, either works if you don't have FINAL_DATA, so why attempt to fix implementations.